aioftp/client.py

```diff
@@ -120,7 +120,8 @@ def __init__(self,
                  read_speed_limit=None,
                  write_speed_limit=None,
                  path_timeout=None,
-                 path_io_factory=pathio.PathIO):
+                 path_io_factory=pathio.PathIO,
+                 list_compatibility_level=2):
```

Comment:
This definitely should be removed, since user can disconnect from current server and connect to another, but `list_compatibility_level` already triggered to `LIST`.